content/en/docs/reference/glossary/events.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Events
+id: k8s-events
+date: 2019-08-30
+full_link: /docs/concepts/cluster-administration/events/
+short_description: >
+  Events are objects that provide insight into what is happening inside a {{< glossary_tooltip text="cluster" term_id="cluster" >}}.
+
+aka: 
+tags:
+- architecture
+- fundamental
+---
+ Kubernetes events are objects that provide insight into what is happening inside a {{< glossary_tooltip text="cluster " term_id="cluster" >}}, such as what decisions were made by the scheduler or why some pods were evicted from the node.
```

Comment:
```suggestion
In Kubernetes, Events are objects that provide insight into what is happening inside a {{< glossary_tooltip text="cluster " term_id="cluster" >}}, such as what decisions were made by the scheduler or why some Pods were evicted from a Node.
```
(see [Use camel case for API objects](https://kubernetes.io/docs/contribute/style/style-guide/#use-camel-case-for-api-objects))

Reply:
Ooops - corrected!